sourcemash/api/categories.py

```diff
@@ -1,11 +1,20 @@
 from . import api, login_required
-from flask.ext.restful import Resource, marshal, fields
+from sourcemash.database import db
+from flask.ext.restful import Resource, marshal, fields, reqparse, inputs
 from flask.ext.security import current_user
 
-from sourcemash.models import Item, Category, UserItem
+from sourcemash.models import Item, Category, UserItem, UserCategory
 
 from sqlalchemy import func
 
+class isUnread(fields.Raw):
+    def output(self, key, category):
+        try:
+            unread = UserCategory.query.filter_by(user=current_user, category_id=category.id).one().unread
+        except:
```

Comment:
Let's specify this except, like seen here: http://stackoverflow.com/questions/18110033/getting-first-row-from-sqlalchemy